File: Backend/wearService/bo/tag.py
class Tag:

    def __init__(self):
        self._tagID = 0
        self._tagKey = 0
        self._tagFree = 0

    def setTagID(self,value):
        self._tagID = value

    def getTagID(self):
        return self._tagID

    def setTagKey(self,value):
        self._tagKey = value;

    def getTagKey(self):
        return self._tagKey

    def getTagFree(self):
        return self._tagFree

    def setTagFree(self, value):
        self._tagFree = value;


    def __str__(self):
        print(self._tagID,self._tagKey, self._tagFree)

File: Apps/Mobil (Android)/app/src/main/java/com/example/bo/Tag.java
package com.example.bo;

import java.math.BigInteger;

public class Tag {

    private long tagID;
    private int tagFree;
    private String tagKey;

    public Tag() {
    }
    public Tag(long tagID, int tagFree, String tagKey) {
        this.tagID = tagID;
        this.tagFree = tagFree;
        this.tagKey = tagKey;
    }

    public long getTagID() {
        return tagID;
    }

    public void setTagID(long tagID) {
        this.tagID = tagID;
    }

    public void setTagFree(int tagFree){
        this.tagFree = tagFree;
    }

    public int getTagFree(){
        return tagFree;
    }

    public void setTagKey(String tagKey){
        this.tagKey = tagKey;
    }

    public String getTagKey() {
        return tagKey;
    }

    public String toString(){
        return this.tagKey;
    }


}